src/libs/actions/PersonalDetails.js

```diff
@@ -337,6 +337,56 @@ function updateDisplayName(firstName, lastName) {
     Navigation.navigate(ROUTES.SETTINGS_PROFILE);
 }
 
+/**
+ * Updates timezone's 'automatic' setting, and updates
+ * selected timezone if set to automatically update.
+ *
+ * @param {Object} timezone
+ * @param {Boolean} timezone.automatic
+ * @param {String} timezone.selected
+ */
+function updateAutomaticTimezone(timezone) {
+    API.write('UpdateAutomaticTimezone', {
+        timezone: JSON.stringify(timezone),
+    }, {
+        optimisticData: [{
```

Comment:
Any reason not to include `failureData` for these two commands?

Reply:
Hey @NikkiWines ! It's because there is a decision [in the design doc](https://docs.google.com/document/d/17akqrbC-w8FqEoDLvcLmrX6iw4uaVrT_xlwAl-IhEzE/edit#heading=h.d016bc22cukl) that these actions should follow the [Offline Pattern A](https://github.com/Expensify/App/blob/main/contributingGuides/OFFLINE_UX.md#a---optimistic-without-feedback-pattern).